third_party/pyth/price-service/src/promClient.ts

```diff
@@ -70,31 +72,25 @@ export class PromClient {
     this.receivedVaaCounter.inc();
   }
 
-  addResponseTime(path: string, status: number, duration: DurationInMs) {
+  addPriceUpdatesGap(gap: DurationInSec) {
+    this.priceUpdatesGapHistogram.observe(gap);
+  }
+
+  // We have multiple paths and for the time being it is not important for us
+  // to capture it. We might consider capturing it in the future.
```

Comment:
this confuses me. Seems like knowing which API method is queried is important? Can you aggregate these series in datadog instead of collapsing the information here?

Reply:
Ah I think we should not capture it when we don't use it anywhere, we'll add it when we really need it. I can remove the parameter too. I think currently we don't need it, but if we add more endpoints with business logic we probably need to.

And yeah, we can aggregate the metric when we scrape them, but I would like to keep all the logic in the code. It's not blockchain here and YAGNI principle is true 😀